### ISSUE TITLE
The pretrained model "crowdhuman_final.pth" not available

### ISSUE THREAD

**Issue by maurus95** [PeizeSun/TransTrack#85]
The link of the pretrained weights file [crowdhuman_final.pth](https://drive.google.com/drive/folders/1DjPL8xWoXDASrxgsA3O06EspJRdUXFQ-?usp=sharing) is not found by the server (Error 404). Can you provide a new link to the pretrained model file?

**Comment by maurus95** [comment]
Nevermind, I found the alternative links in the [README.md](https://github.com/PeizeSun/TransTrack/blob/main/README.md#validation-set) in the section "Validation Set".
